DQMOffline/Trigger/interface/HLTTauDQMOfflineSource.h

```diff
@@ -55,6 +58,10 @@ class HLTTauDQMOfflineSource : public DQMEDAnalyzer {
       edm::EDGetTokenT<LVColl> token;
     };
     std::vector<RefObject> refObjects_;
+    std::vector<edm::ParameterSet> tagAndProbePaths;
```

Comment:
The size of tagAndProbePaths is used in  HLTTauDQMOfflineSource::dqmBeginRun to check if the original dqm should be done or not. So a member is needed, but it can be boolean. Do you prefer boolean, or shall I leave the code as is?